problemi/boltzmannDistribution/src/boltzmann.cpp

```diff
@@ -0,0 +1,39 @@
+#include "boltzmann/boltzmann.h"
+
+#include <algorithm>
+#include <cmath>
+
+BoltzmannSimulator::BoltzmannSimulator(int molecules, double kBT, double hparam,
+                                       unsigned seed)
+    : molecules_{molecules},
+      deltaU_{kBT / hparam},
+      energies_(molecules, 1.5 * kBT),
+      rng_{seed ? seed : std::random_device{}()},
```

Comment:
un seed uguale a zero credo sia accettabile. in ogni caso forse e' preferibile passare direttamente l'engine